robot.py

```
@@ -159,7 +161,7 @@ def open_robot_session():
     intents = botpy.Intents.all()  # 对目前已支持的所有事件进行监听
     client = MyClient(intents=intents, timeout=60)
 
-    # 更新每日排行榜
-    threading.Thread(target=daily_sched_thread, args=[]).start()
+    # 启动定时任务调度器（具体任务由 activate_scheduled_jobs 在 on_ready 中注册）
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)